File: 1833-find-the-highest-altitude/find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        #Mehnaz
        current = 0
        max_gain = 0
        for g in gain:
            current += g
            max_gain = max(max_gain,current)
        return max_gain
        

        """
        :type gain: List[int]
        :rtype: int
        """
        

File: 1833-find-the-highest-altitude/find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) 
    {   //Mehnaz
        int current_gain = 0;
        int max_gain = 0;
        for(int x : gain)
        {
            current_gain += x;
            max_gain= Math.max(max_gain,current_gain);
        }
        return max_gain;
    }
}